Makefile

```diff
@@ -85,105 +85,69 @@ test:
 .PHONY: test-cover
 test-cover:
 	for TEST in $(TEST_PKGS) ; do \
-    	go test $(BUILD_TAGS) -timeout $(TEST_TIMEOUT)s -cover $(COVER_ARGS) $$TEST ; \
-    done
+		go test $(BUILD_TAGS) -timeout $(TEST_TIMEOUT)s -cover $(COVER_ARGS) $$TEST ; \
+	done
 
 .PHONY: test-race
 test-race:
 	for TEST in $(TEST_PKGS) ; do \
-    	go test $(BUILD_TAGS) -timeout $(TEST_TIMEOUT)s -race $(COVER_ARGS) $$TEST ; \
-    done
+		go test $(BUILD_TAGS) -timeout $(TEST_TIMEOUT)s -race $$TEST ; \
+	done
 
 .PHONY: test-short
 test-short:
-	go test $(BUILD_TAGS) -timeout 60s -short $(COVER_ARGS) ./...
-
-.PHONY: update-bson-corpus-tests
```

Comment:
I don't think any of us use these `update-[spec]-tests` `make` targets. Updating spec tests for an entire component is rarely useful and can still be done with the `update_spec_tests.sh` command if needed.